src/mono/mono/eventpipe/ep-event-source.c

```diff
@@ -14,21 +14,21 @@
 #include "ep-rt.h"
 
 #if defined(HOST_WINDOWS) || defined(HOST_WIN32)
-const ep_char8_t* _ep_os_info = "windows";
+const ep_char8_t* _ep_os_info = "Windows";
 #elif defined(HOST_DARWIN)
-const ep_char8_t* _ep_os_info = "osx";
+const ep_char8_t* _ep_os_info = "macOS";
```

Comment:
i'm not sure if we talked about this during the eventpipe PR.  `HOST_DARWIN` is all apple OSes in Mono.  I think you want to check for `TARGET_OSX` to specifically check for macOS.  

(Yes there's host/target confusion.  but this isn't the AOT compiler part of Mono, so host==target will be true here, I think).

Reply:
OK, I see in coreclr that they use __APPLE__ so maybe I use that as a fallback after all other Apple specific OSes.